src/App.vue

```diff
@@ -7,13 +7,106 @@
 </template>
 
 <script>
+import { workflowService } from 'workflow-service'
+
 const defaultLayout = 'empty'
 
+const QUERIES = {
+  root: `
+    {
+      workflows {
```

Comment:
BTW: I think we will need to improve the whole `gproxy` thing so it can merge this query properly, my bad I should have opened a follow on issue at the time.

Reply:
There are two or three issues created that involve changes in the `gproxy`, so there's room for improvement, but it was a life saver. Without that early in the project, we would have a lot built on top of - probably - multiple queries spread over the project.

Reply:
@oliver-sanders - ditto this - is an issue needed, it will it be obvious later on?